staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -944,8 +944,9 @@ type StorageMedium string
 
 const (
 	StorageMediumDefault   StorageMedium = ""          // use whatever the default is for the node
-	StorageMediumMemory    StorageMedium = "Memory"    // use memory (tmpfs)
+	StorageMediumMemory    StorageMedium = "Memory"    // use memory (e.g. tmpfs on linux)
 	StorageMediumHugepages StorageMedium = "HugePages" // use hugepages
+	StorageMediumUnknown   StorageMedium = "Unknown"   // assume anything we don't explicitly handle is this
```

Comment:
this is awkward as a separate pr will have this fail validation.

Reply:
Do you mean PR #52936? It just validate by `validStorageMedia` which doesn't include `StorageMediumUnknown`.